install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -71,7 +168,11 @@ spec:
         istio-mixer-type: {{ $mname }}
       annotations:
         sidecar.istio.io/inject: "false"
-{{- template "mixer_containers" $ }}
+{{- if eq $mname "policy"}}
+{{- template "policy_container" $ }}
```

Comment:
nit: `template "$mname_container"`

Reply:
I tried to make this work in any number of other ways. Each time, I was blocked by the templating system.

Reply:
Sure.